src/statmodels.jl

```diff
@@ -165,7 +165,8 @@ dof_residual(obj::RegressionModel) = error("dof_residual is not defined for $(ty
 """
     params(model)
 
-Return all parameters of a model.  For a linear model, this would return coefficients as well as any other parameters.
+Return all parameters of a model.  For a linear model, this would return coefficients as
```

Comment:
I still don't like that explanation. How about something like this? To be improved if somebody else has suggestions.
```
Return all parameters of a model. Contrary to [`coef`](@ref), this includes
parameters other than those corresponding to the effect of independent variables
(e.g. in a linear model, those which determine the linear predictor).
```

Or we could add that mention to `coef` instead, pointing to `params` to get all parameters. In any case, it makes sense to have a link from each of the docstrings to the other.